config/config.go

```diff
@@ -204,7 +204,7 @@ type Config struct {
 	RuleFiles      []string        `yaml:"rule_files,omitempty"`
 	ScrapeConfigs  []*ScrapeConfig `yaml:"scrape_configs,omitempty"`
 
-	RemoteWriteConfig RemoteWriteConfig `yaml:"remote_write,omitempty"`
```

Comment:
From a practical standpoint we already have users confused about basic YAML syntax. remote.read is going to be harder to get right than remote_read. It'll also be easier to talk about with users in terminology terms (e.g. alert_relabel_configs isn't called relabel_configs for this reason).